06_gpu_and_ml/stable_diffusion_aitemplate/stable_diffusion_aitemplate.py

```diff
@@ -0,0 +1,234 @@
+# # Stable Diffusion (AITemplate Edition)
+#
+# This example shows the Stable Diffusion 1.5 compiled with [AITemplate](https://github.com/facebookincubator/AITemplate) to run faster on Modal.
+# There is also a [Stable Diffusion CLI example](/docs/guide/ex/stable_diffusion_cli).
+# 
+# #### Upsides
+#  - Image generation improves over the CLI example to about 550ms per image generated (A10G, 10 steps, 512x512, png).
+# 
+# #### Downsides
+#  - Width and height as well as batch size must be configured prior to compilation which takes about 15 minutes.
+#  - In this example the compilation is done at docker image creation.
+#  - Cold start time are also increased to upto ~30s.
+
+# ## Setup
+import io
+import os
+import sys
+import modal
+from fastapi import FastAPI, Response
+from pydantic import BaseModel
+
+hf_dir = "/root/.cache/huggingface"
+hf_volume = modal.SharedVolume().persist("hf-cache-vol")
+
+# ## Define the GPU Type and model, dimensions and batch size
+gpu_type = "A10G"
+model_id = "runwayml/stable-diffusion-v1-5"
+width = 512
+height = 512
+batch_size = 1
+
+# ## Set the paths to the AITemplate example. This is needed to import the compiler and pipeline.
+
+def set_paths():
+    ait_sd_example_path = "/app/AITemplate/examples/05_stable_diffusion"
```

Comment:
I see this is the same path ref'd at https://github.com/modal-labs/modal-examples/pull/308/files#diff-40fda8df91885f0f4c10fdd86e8c1f827316b8da25b0b944f7033f180deb6cb3R161

Could break it out into a `workdir` global config variable?